api/opencl_platform_layer.txt

```diff
@@ -1385,7 +1386,7 @@ on one or more devices specified in the context.
   | cl_bool
       | Specifies whether the user is responsible for synchronization
         between OpenCL and other APIs.
-        Please refer to the specific sections in the OpenCL 2.0 extension
+        Please refer to the specific sections in the OpenCL 2.2 extension
```

Comment:
Should we start referring to the unified specification instead?

Reply:
Yes, I'll reword this.  Those the unified specification is still v2.2-10.

Reply:
Done, I know just say "OpenCL Extension Specification".